packages/commonwealth/client/scripts/views/pages/discussions/DiscussionsPage.tsx

```diff
@@ -165,7 +165,13 @@ const DiscussionsPage = ({ topicName }: DiscussionsPageProps) => {
               stage={stageName}
               featuredFilter={featuredFilter}
               dateRange={dateRange}
-              totalThreadCount={threads ? totalThreadsInCommunity : 0}
+              totalThreadCount={
+                isOnArchivePage
```

Comment:
@mzparacha 
I see what you're doing, but some parentheses or a different alignment might make this more readable, e.g. 

```js
isOnArchivePage ? filteredThreads.length || 0 : 
threads ? totalThreadsInCommunity : 
0
```

Reply:
prettier will reformat it anyways

Reply:
same as @masvelio, tried to manually format with prettier but no changes. @Miaplacidus plz lmk if GTG

Reply:
I'm declaring this g2g, if prettier accepts it then that's that.